test/e2e/deploy_native_test.go

```diff
@@ -50,10 +50,10 @@ func TestDeploymentOfNativeContract(t *testing.T) {
 			WithMethod(primitives.ContractName(fmt.Sprintf("CounterFrom%d", counterStart)), "get")
 
 		printTestTime(t, "call method - start", &lt)
-		response, err := h.callMethod(getCounter.Builder())
+		response, err2 := h.callMethod(getCounter.Builder())
 		printTestTime(t, "call method - end", &lt)
 
-		if err == nil && response.CallMethodResult() == protocol.EXECUTION_RESULT_SUCCESS {
+		if err2 == nil && response.CallMethodResult() == protocol.EXECUTION_RESULT_SUCCESS {
```

Comment:
err2?

Reply:
1) test
2) overshadow

Reply:
again, overshadowing is a smell that variables are reassigned. The previous `err` needs to be scoped.